packages/web/src/scss/components/Header/index.html

```diff
@@ -2,17 +2,65 @@
-          width="65"
-          height="24"
-          alt="Spirit"
-        />
+    <header class="Header">
+      <a href="#" aria-label="JobBoard homepage">
+        <div class="ProductLogo">
+          <svg width="178" height="44" viewBox="0 0 178 44" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
maybe we can either use `{{setVar "logo" "<svg...` to store the logo or add it to global Vite config runtimeOptions data and print it from there?

What do you think?